arenadata/Dockerfile

```
@@ -1,5 +1,10 @@
 FROM rockylinux:8.8 as base
 
+# Allow usage of custom compilers, however, gcc is the default
+ARG CC=gcc
+ARG CXX=g++
+ARG SKIP_UNITTESTS=false
```

why not
```suggestion
ARG SKIP_UNITTESTS=
```
and unchanged below?

---

compile_gpdb.bash checks that SKIP_UNITTESTS is not empty, but doesn't check its value. If we leave it as is, what would happen when SKIP_UNITTESTS is set to false manually via --build-arg? I suppose tests still will be skipped though an opposite is expected

---

Why do you assume that the value of this variable is necessarily boolean and necessarily in lowercase? Why can't I set 0/1 or on/off or yes/no? Previously, it was assumed that if any value was set, no matter what, then unit-tests should not be performed.

---

> Why do you assume that the value of this variable is necessarily boolean and necessarily in lowercase? Previously, it was assumed that if any value was set, no matter what, then unit-tests should not be performed.

Sound good. It gets rid of all confusion.

---

> Why do you assume that the value of this variable is necessarily boolean and necessarily in lowercase?

I don't assume, I explicitly changed the logic to expect a value so there won't be confusions when SKIP_UNITTESTS is set to false, but tests run anyway. Why make it complex and make someone debug docker container when it behaves not as expected?

---

I suggest use `ARG SKIP_UNITTESTS=` and also document its using.

---

Why omit default value? I don't understand what's the profit. Why do you think that non-obvious behavior is preferable?

---

Done